notebooks/tr/agents.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<ul><li>çeviren olarak kendi ismini Aymeric'in yanına ekleyebilirsin :)</li><li>Transformers Agents* olarak kalabilir kütüphanenin adı olduğu için</li><li>Bağımlılık yerine kütüphane diyebiliriz</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>